devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -464,6 +482,12 @@ private Boolean secondsPast(timestamp, seconds) {
 	return (new Date().time - timestamp) > (seconds * 1000)
 }
 
+def calculateLength(int alarmLength){
```

Comment:
Please make the function name more descriptive, such as `calculateAlarmLength` or `calcAlarmLength` or even `calcEverspringAlarmLen` or some derivative of this is meant to be specific to Everspring. Also, please add a space between `)` and `{`.

Reply:
I would second @dkirker's comment. This function name should indicate somehow that it's specific to Everspring.